File: src/main/java/com/brasil/transparente/api/repository/MinisterioRepository.java
package com.brasil.transparente.api.repository;

import com.brasil.transparente.api.entity.Ministerio;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface MinisterioRepository extends JpaRepository<Ministerio, String> {

    List<Ministerio> findByPoderPoderId(Long poderId);

}

File: src/main/kotlin/com/brasil/transparente/api/repository/MinisterioRepository.kt
package com.brasil.transparente.api.repository

import com.brasil.transparente.api.entity.Ministerio
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface MinisterioRepository : JpaRepository<Ministerio, String> {

    fun findByPoderPoderId(poderId : Long) : List<Ministerio>

}